test/requirements.yml

```diff
@@ -0,0 +1,3 @@
+---
+- src: chrismeyersfsu.provision_docker
```

Comment:
why are we not using [docker_container](http://docs.ansible.com/ansible/docker_container_module.html#docker-container)?